api/lds.proto

```diff
@@ -4,6 +4,8 @@
 
 syntax = "proto3";
 
+package envoy.api.v1beta1;
```

Comment:
I think this should be `v2beta1`, the existing endpoints have v1 in them, see https://lyft.github.io/envoy/docs/configuration/http_conn_man/rds.html. @mattklein123